Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/UpdateTextAccessibilityController.java

```diff
@@ -0,0 +1,32 @@
+package edu.asu.spring.quadriga.web.workspace;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.domain.enums.ETextAccessibility;
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+import edu.asu.spring.quadriga.exceptions.FileStorageException;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.textfile.ITextFileManager;
+
+@Controller
+public class UpdateTextAccessibilityController {
+
+    @Autowired
+    private ITextFileManager tfManager;
+
+    @RequestMapping(value = "/auth/workbench/workspace/{workspaceId}/{textId}/{accessStatus}/updateAccessibility", method = RequestMethod.GET)
```

Comment:
by definition, put is usually used to store (or update) an entity at a given url:
"The PUT method requests that the enclosed entity be stored under the supplied Request-URI. If the Request-URI refers to an already existing resource, the enclosed entity SHOULD be considered as a modified version of the one residing on the origin server."

Post basically just says, do something with this:
"The actual function performed by the POST method is determined by the server and is usually dependent on the Request-URI. [...] The action performed by the POST method might not result in a resource that can be identified by a URI. "

If we consider our text file the resource to be updated, then post would be wrong because we're just updating one field here (and we are only sending data for one field). PUT requires that if I send the same response again, I get the same result (in the sense of a resource => text file would be exactly the same). This would not be true because our text file might have changed in between, so although the accessibility would be the same, the text file itself might be different. 

Does this make sense?